app/dashboard/src/components/AriaComponents/Inputs/variants.ts

```diff
@@ -9,32 +9,33 @@ import { TEXT_STYLE } from '../Text'
 export const INPUT_STYLES = tv({
   base: 'block w-full overflow-hidden bg-transparent transition-[border-color,outline] duration-200',
   variants: {
+    // All variants SHOULD use objects, otherwise extending from them does not work.
     disabled: {
       true: { base: 'cursor-default opacity-50', textArea: 'cursor-default' },
       false: { base: 'cursor-text', textArea: 'cursor-text' },
     },
     invalid: {
       // Specified in compoundVariants. Real classes depend on Variants
-      true: '',
+      true: { base: '' },
     },
     readOnly: {
-      true: 'cursor-default',
-      false: 'cursor-text',
+      true: { base: 'cursor-default' },
+      false: { base: 'cursor-text' },
     },
     size: {
       medium: { base: 'px-[11px] pb-[6.5px] pt-[8.5px]', icon: 'size-4' },
       small: { base: 'px-[11px] pb-0.5 pt-1', icon: 'size-3' },
       custom: {},
     },
     rounded: {
-      none: 'rounded-none',
-      small: 'rounded-sm',
-      medium: 'rounded-md',
-      large: 'rounded-lg',
-      xlarge: 'rounded-xl',
-      xxlarge: 'rounded-2xl',
-      xxxlarge: 'rounded-3xl',
-      full: 'rounded-full',
+      none: { base: 'rounded-none' },
```

Comment:
looks like leftover? On the first glance this changes nothing?

Reply:
see comment added further above: "All variants SHOULD use objects, otherwise extending from them does not work."